src/QMCDrivers/WFOpt/GradientTestInput.h

```diff
@@ -25,9 +25,11 @@ class GradientTestInput
 
 protected:
   bool do_param_output_ = false;
+  double finite_diff_delta_ = 1e-5;
 
 public:
   bool do_param_output() { return do_param_output_; }
+  double finite_diff_delta() { return finite_diff_delta_; }
```

Comment:
The overall convention for function naming is starting with a verb. `get_finite_diff_delta` ? also both member functions should be marked const.